packages/backend/src/plugins/onboarding.ts

```diff
@@ -0,0 +1,16 @@
+import { createRouter } from 'backstage-plugin-dask-onboarding-backend';
+import { Router } from 'express';
+import { PluginEnvironment } from '../types';
+
+export default async function createPlugin(
```

Comment:
can we rename this file to something more specific, ie `dask-onboarding.ts` for example?

Reply:
👍